Strange log in https://dev.azure.com/openupm/openupm/_build/results?buildId=39143&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=f8ed7bd8-2a7f-56f6-9385-7fc29a8b5b7b
```
Starting: Prepare context variables
==============================================================================
Task         : Command line
Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
Version      : 2.231.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
==============================================================================
Generating script.
========================== Starting Command Output ===========================
/usr/bin/bash --noprofile --norc /home/vsts/work/_temp/c423f010-ffdd-453c-8479-61d229331c45.sh
repoUrl:
repoBranch:
packageName:
packageVersion:
variable repoUrl not found

##[error]Bash exited with code '1'.
Finishing: Prepare context variables
```
I suppose, first of all need to pay attention on this thing.

--- a/findPackage.js
+++ b/findPackage.js
@@ -23,7 +23,10 @@ const findPackage = function(packageName, searchPath) {
         } catch (err) {
           return;
         }
-        if (pkg.name != packageName) return;
+        if (pkg.name != packageName) {
+            console.log(`wrong package name in ${path.join(dirname, basename)}: actual=${pkg.name}, expected=${packageName}`);
+            return;
+        }
         finder.stop();
         resolve({ dirname, pkg, file });
       }

